internal/server/v1/dlq/service.go

```diff
@@ -24,3 +28,16 @@ func NewService(client entropyv1beta1rpc.ResourceServiceClient, gcsClient gcs.Bl
 		cfg:       cfg,
 	}
 }
+
+func (s *Service) getDlqJob(ctx context.Context, jobURN string) (*models.DlqJob, error) {
+	res, err := s.client.GetResource(ctx, &entropyv1beta1.GetResourceRequest{Urn: jobURN})
+	if err != nil {
+		return nil, err
+	}
+
+	dlqJob, err := MapToDlqJob(res.GetResource())
+	if err != nil {
+		return nil, err
```

Comment:
always try to wrap the error, we can do it like
```suggestion
		return nil, fmt.Errorf("error mapping to dlq job: %w", err)
```

Reply:
do we need to do this everywhere or just inside the services?

Reply:
in most cases we need it